File: Golang/Richest Customer Wealth.go
func maximumWealth(accounts [][]int) int {
    maxWealth := 0
    for i:=0; i < len(accounts);i++ {
        temp := 0
        for j:=0; j < len(accounts[i]);j++{
            temp = temp + accounts[i][j]
        }
        if temp > maxWealth {
            maxWealth = temp
        }
    }
    return maxWealth
}

File: Python3/Richest Customer Wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        maxWealth = 0

        for i in range(0, len(accounts)):
            temp = 0
            for j in range(0, len(accounts[i])):
                temp = temp + accounts[i][j]
            if temp > maxWealth:
                maxWealth = temp
        
        return maxWealth